modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/ParameterModelValidator.java

```diff
@@ -191,4 +181,46 @@ private void validateNameCollisionWithTypes(ParameterModel parameterModel, Strin
 
     }
 
-}
+    private void validateParameterGroup(ParameterModel parameterModel, String ownerName, String ownerModelType, String extensionName)
+    {
+        ParameterGroupModelProperty parameterGroup = parameterModel
+                .getModelProperty(ParameterGroupModelProperty.class)
+                .orElse(null);
```

Comment:
don't do this and check for null later, it's already an optional, go for `ifPresent`